website/oauth/models/__init__.py

```diff
@@ -489,7 +491,7 @@ class ApiOAuth2Application(StoredObject):
     name = fields.StringField(index=True, required=True, validate=[string_required, MaxLengthValidator(200)])
     description = fields.StringField(required=False, validate=MaxLengthValidator(1000))
 
-    date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow,
+    date_created = fields.DateTimeField(auto_now_add=timezone.now,
```

Comment:
You can get around django's model validation a few ways:

- Use `bulk_create` or `bulk_update`, these don't run any of the signals or validation on the models.
- Use straight SQL and go around the ORM. This bypasses the ORM completely and won't run any validation other than making sure things pass the constraints that are actually in the database (unique, not null, type (varchar, int, jsonb), etc)

There might be others that I don't know about or aren't thinking of right now. Those are the main ones though. So there's always another way. Even if the constraint was in SQL we could drop the constraint at the beginning of the migration, do our dirt, and then put it back on at the end.